test/onnx/test_operators.py

```diff
@@ -454,11 +465,17 @@ def forward(self, x, y):
 
 
 if __name__ == '__main__':
+    no_onnx_dep_flag = '--no-onnx'
+    _onnx_dep = no_onnx_dep_flag not in common.UNITTEST_ARGS
+    if no_onnx_dep_flag in common.UNITTEST_ARGS:
+        common.UNITTEST_ARGS.remove(no_onnx_dep_flag)
     onnx_test_flag = '--onnx-test'
     _onnx_test = onnx_test_flag in common.UNITTEST_ARGS
     if onnx_test_flag in common.UNITTEST_ARGS:
         common.UNITTEST_ARGS.remove(onnx_test_flag)
     if _onnx_test:
+        _onnx_dep = True
+        import test_onnx_common
```

Comment:
why do you need import at this point?

Reply:
Because we need to remove the old test cases.